src/pybind/mgr/dashboard/services/cephfs.py

```diff
@@ -49,13 +49,14 @@ def get_dir_list(self, dirpath, level):
             dent = self.cfs.readdir(d)
             paths = [dirpath]
             while dent:
-                logger.debug("[CephFS] found entry=%s", dent.d_name)
-                if dent.d_name in ['.', '..']:
+                d_name = dent.d_name.decode('utf-8')
+                logger.debug("[CephFS] found entry=%s", d_name)
+                if d_name in ['.', '..']:
                     dent = self.cfs.readdir(d)
                     continue
                 if dent.is_dir():
-                    logger.debug("[CephFS] found dir=%s", dent.d_name)
-                    subdirpath = '{}{}/'.format(dirpath, dent.d_name)
+                    logger.debug("[CephFS] found dir=%s", d_name)
+                    subdirpath = '{}{}/'.format(dirpath, d_name)
```

Comment:
`b'{}{}/'` instead

Reply:
That does not work ``AttributeError: 'bytes' object has no attribute 'format'`` but will find a solution.

Reply:
Done